docs/getting_started/quickstart.md

```diff
@@ -136,7 +125,7 @@ $ curl -v localhost:8000/v2/health/ready
 The HTTP request returns status 200 if Triton is ready and non-200 if
 it is not ready.
 
-## Getting The Client Examples
+## Sending an Infernce Request
```

Comment:
```suggestion
## Sending an Inference Request
```

Reply:
Heads up that while this was marked as resolved, I don't see the correction reflected on my end.